src/Files.App/MainWindow.xaml.cs

```diff
@@ -339,5 +343,28 @@ x.tabItem.NavigationParameter.NavigationParameter is PaneNavigationArguments pan
 				}
 			}
 		}
+
+        public bool SetCanWindowToFront(bool canWindowToFront)
+        {
+            lock (_canWindowToFrontLock)
+            {
+                if (CanWindowToFront != canWindowToFront)
+                {
+                    CanWindowToFront = canWindowToFront;
+                    return true;
+                }
+                return false;
+            }
+        }
```

Comment:
Use tabs for indentation.
```suggestion
		public bool SetCanWindowToFront(bool canWindowToFront)
		{
			lock (_canWindowToFrontLock)
			{
				if (CanWindowToFront != canWindowToFront)
				{
					CanWindowToFront = canWindowToFront;
					return true;
				}
				return false;
			}
		}
```